src/biogeochem/SatellitePhenologyMod.F90

```diff
@@ -410,12 +410,12 @@ subroutine SatellitePhenology(bounds, num_nolakep, filter_nolakep, &
             ol = min( max(snow_depth(c)-hbot(p), 0._r8), htop(p)-hbot(p))
             fb = 1._r8 - ol / max(1.e-06_r8, htop(p)-hbot(p))
          else
-            fb = 1._r8 - max(min(snow_depth(c),0.2_r8),0._r8)/0.2_r8   ! 0.2m is assumed
-            !depth of snow required for complete burial of grasses
+            fb = 1._r8 - (max(min(snow_depth(c),max(0.05,htop(p)*0.8_r8)),0._r8)/(max(0.05,htop(p)*0.8_r8)))
          endif
 
          ! area weight by snow covered fraction
-
+         !grass and crop snow burial changes with PFT height
+         !accounts for a 20% bending factor, as used in Lombardozzi et al. (2018) GRL 45(18), 9889-9897
```

Comment:
- [ ] I think this comment should be a few lines up, where you have changed fb (line 413)